sim_pipeline/Deflector/OM10AGNlens.py

```diff
@@ -0,0 +1,24 @@
+from gg_lens import GGLens
+from sim_pipeline.Deflector.deflector_base import LensBase
```

Comment:
is this DeflectorBase or LensBase? I don't see LensBase in the pull request

Reply:
DeflectorBase, that was a residual from a name change. Thanks!